janitor/timeseries.py

```diff
@@ -275,88 +294,72 @@ def flag_jumps(
     Create boolean column(s) that flag whether or not the change
     between consecutive rows exceeds a provided threshold.
 
-    Functional usage example:
-
-    ```python
-    import pandas as pd
-    import janitor.timeseries
-
-    df = pd.DataFrame(...)
-
-    df = flag_jumps(
-        df=df,
-        scale="absolute",
-        direction="any",
-        threshold=2,
-    )
-    ```
-
     Method chaining example:
 
     ```python
-    import pandas as pd
-    import janitor.timeseries
-
-    df = (
-        pd.DatFrame(...)
-        .flag_jumps(
-            scale="absolute",
-            direction="any",
-            threshold=2,
-        )
-    )
-    ```
-
-    Detailed chaining examples:
-
-    ```python
-    # Applies specified criteria across all columns of the DataFrame
-    # Appends a flag column for each column in the DataFrame
-    df = (
-        pd.DataFrame(...)
-        .flag_jumps(
-            scale="absolute",
-            direction="any",
-            threshold=2
-        )
-    )
+    >>> import pandas as pd
+    >>> import janitor.timeseries
+
+    >>> ts_index = pd.date_range("1/1/2019", periods=3, freq="1H")
+    >>> test_df = pd.DataFrame(
+    ...    {
+    ...         "col1": [*range(3)],
+    ...         "col2": [*range(0,6,2)],
+    ...         "col3": [20, 21, 42],
+    ...    },
+    ...    index=ts_index,
+    ... )
+    >>> test_df
+                         col1  col2  col3
+    2019-01-01 00:00:00     0     0    20
+    2019-01-01 01:00:00     1     2    21
+    2019-01-01 02:00:00     2     4    42
+    >>> test_df.flag_jumps(
+    ...        scale="absolute",
+    ...        direction="any",
+    ...        threshold=1.0,
+    ...    )
```

Comment:
Similar comments here about the indentation, the section block header and you may remove the "```python" code block marker.